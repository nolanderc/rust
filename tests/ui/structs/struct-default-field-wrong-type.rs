struct S {
    foo: u32 = false, //~ ERROR mismatched types [E0308]
}

fn main() {
    let s = S{ foo: 32 };
}
