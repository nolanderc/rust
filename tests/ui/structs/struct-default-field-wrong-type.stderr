error[E0308]: mismatched types
  --> $DIR/struct-default-field-wrong-type.rs:2:16
   |
LL |     foo: u32 = false,
   |                ^^^^^ expected `u32`, found `bool`

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0308`.
